File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kinh nghiệm coder/CODEREXP.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int Basic, Medium, Extra, Hard;
    cin >> Basic >> Medium >> Extra >> Hard;
    int Point = Basic * 10 + Medium * 20 + Extra * 30 + Hard * 40;
    cout << Point << endl;
    if (Point >= 7000)
        cout << "Coder Giao Su";
    else if (Point >= 5500)
        cout << "Coder Tien Si";
    else if (Point >= 4200)
        cout << "Coder Thac Si";
    else if (Point >= 3500)
        cout << "Coder Dai Hoc";
    else if (Point >= 2500)
        cout << "Coder Cao Dang";
    else if (Point >= 2000)
        cout << "Coder Trung Cap";
    else if (Point >= 1500)
        cout << "Coder THPT";
    else if (Point >= 1000)
        cout << "Coder THCS";
    else if (Point >= 500)
        cout << "Coder Tieu Hoc";
    else if (Point >= 100)
        cout << "Coder Lop La";
    else if (Point >= 50)
        cout << "Coder Lop Choi";
    else if (Point >= 1)
        cout << "Coder Lop Mam";
    else
        cout << "Coder So Sinh";
    return 0;
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kinh nghiệm coder/CODEREXP.cs
using System;
namespace Kinhnghiemcoder
{
    public class Program
    {
        static void Main(string[]args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int n = input.Length;
            int[] a = new int[n];
            for(int i = 0; i < n; i++)
                a[i] = int.Parse(input[i]);
            int Basic = a[0];
            int Medium = a[1];
            int Extra = a[2];
            int Hard = a[3];
            int Point = Basic * 10 + Medium * 20 + Extra * 30 + Hard * 40;
            Console.WriteLine(Point);
            if (Point >= 7000)
                Console.Write("Coder Giao Su");
            else if (Point >= 5500)
                Console.Write("Coder Tien Si");
            else if (Point >= 4200)
                Console.Write("Coder Thac Si");
            else if (Point >= 3500)
                Console.Write("Coder Dai Hoc");
            else if (Point >= 2500)
                Console.Write("Coder Cao Dang");
            else if (Point >= 2000)
                Console.Write("Coder Trung Cap");
            else if (Point >= 1500)
                Console.Write("Coder THPT");
            else if (Point >= 1000)
                Console.Write("Coder THCS");
            else if (Point >= 500)
                Console.Write("Coder Tieu Hoc");
            else if (Point >= 100)
                Console.Write("Coder Lop La");
            else if (Point >= 50)
                Console.Write("Coder Lop Choi");
            else if (Point >= 1)
                Console.Write("Coder Lop Mam");
            else
                Console.Write("Coder So Sinh");
        }
    }
}